app/scripts/angular-apimock.js

```diff
@@ -309,10 +324,11 @@ angular.module('apiMock', [])
 		// Expose public interface for provider instance
 		//
 
-		function ApiMock(_$location, _$log, _$q) {
+		function ApiMock(_$location, _$log, _$q, _$filter) {
```

Comment:
Nope. Angular looks at that and injects whatever I want. `$filter` is supported in 1.2 and up.